ui/js/graphs.js

```diff
@@ -468,16 +478,31 @@ perf.controller('GraphsCtrl', [
                              label: series.projectName + " " + series.name,
                              data: [],
                              resultSetData: [],
+                             customeasure:$scope.myMeasure,
                              thSeries: jQuery.extend({}, series)
                            }
                            response.data[0].blob.forEach(function(dataPoint) {
-                             var mean = dataPoint.mean;
```

Comment:
So this block feels partly right, partly not. The part where you insert data into the graph based on the selected measure is correct (though you should just use $scope.myMeasure, instead of defining it inside the flotSeries). The series.inactive and the series.mycolor part I don't think is right though -- the correct approach I think is to just define a `series.active` variable, and key it off of whether $scope.myMeasure and the nature of the series. Something like this:

    // summary series only have mean, so are only active if that measure is selected
    series.active = (!series.subtestSignatures || $scope.myMeasure === 'mean')'

I would do that calculation in the plotGraph function, just before things are displayed (as opposed to here, where we're calculating graph values).